src/Tests/FunctionalTests/ClassActivation/ClassActivation.csproj

```diff
@@ -14,4 +14,10 @@
     <ProjectReference Include="..\..\..\Projections\Windows\Windows.csproj" />
   </ItemGroup>
 
+  <Target Name="TestGatherRefPaths" BeforeTargets="CoreCompile">
```

Comment:
Is this and the target in the other project just a leftover from testing I assume?

Reply:
Yea ignore the test leftover. They won't be in final